### DISCUSSION TITLE
Milvus Flush Data

### DISCUSSION THREAD

**Discussion by wilhart** [milvus-io/milvus#9627] [Q&A and General discussion] [answered]
I'm using Milvus Standalone for testing on Mac.
I want to know the interval when milvus will send data to disk after inserted data loaded to memory.
And what happen if the memory full when inserting?
![Screen Shot 2021-10-11 at 15 03 41](https://user-images.githubusercontent.com/55882199/136754278-fc410541-01a8-474e-b381-38ba88892480.png)

Thank you

**Comment by yhmo** [comment, marked as answer]
Seems you are using milvus v1.x.
For 1.x, default flush interval is 1 second. You can change the value in the server_config.yaml:
```
storage:
  auto_flush_interval: 1
```

In the server_config.yaml, there is a value to control insert buffer size:
```
cache:
  insert_buffer_size: 1GB
```
Default value is 1Gb, if the inserted data exceed this value, the milvus will trigger a flush action.
